.github/pull_request_template.md

```diff
@@ -1,28 +1,32 @@
 [//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"
 
-## What's new in this PR
-### Description
+## :tada: What's new in this PR :tada:
+
+### :cloud: :cloud: Description :cloud: :cloud:
+
 [//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
 
-### Screenshots
-[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy!"
+### Screenshots :calling:
 
-## How to review
-[//]: # "Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?"
+[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! If you are making changes here, please CC: @leexesther at the bottom."
 
-## Next steps
-[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."
+## :zap: How to review :zap:
 
-## Relevant Links
+[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'
 
-### Online sources
-[//]: # "Optional - copy links to any tutorials or documentation that was useful to you when working on this PR"
+## :seedling: Next steps :seedling:
 
-### Related PRs
-[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"
+[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."
+
+## Relevant Links :link:
 
+### Online sources :information_source:
 
+[//]: # 'Optional - copy links to any tutorials or documentation that was useful to you when working on this PR'
 
+### Related PRs :raised_hands:
+
+[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"
+[//]: # 'This tags the project leader as a default. Feel free to change, or add on anyone who you should be in on the conversation.'
```

Comment:
I would add a couple of lines between the "Optional" and the "This tags the PL" comment so it's clear they're supposed to type between the lines